docs/content/connectors/mysql-cdc(ZH).md

```diff
@@ -402,6 +408,7 @@ CREATE TABLE products (
     db_name STRING METADATA FROM 'database_name' VIRTUAL,
     table_name STRING METADATA  FROM 'table_name' VIRTUAL,
     operation_ts TIMESTAMP_LTZ(3) METADATA FROM 'op_ts' VIRTUAL,
+    operation STRING METADATA FROM 'op' VIRTUAL,
```

Comment:
We have introduced `op_ts` column, I prefer `op_type` for the new metadata column name, WDYT @whhe ?

Reply:
+1, op_type is better.

Reply:
Thanks @whhe for the quick response, I'll raise a PR to improve this.